.env.example

```
@@ -86,11 +86,25 @@ DEFAULT_AI_ASSISTANT=claude
 # TITLE_GENERATION_MODEL=haiku
 
-# GitHub Token (for GitHub CLI and commands)
-GH_TOKEN=
-GITHUB_TOKEN=  # Same as GH_TOKEN, used by adapter
+# ---- GitHub: choose ONE auth mode for the bot adapter ----
+# Archon refuses to start if BOTH GITHUB_TOKEN and GITHUB_APP_ID are set.
 
-# GitHub Webhooks
+# PAT mode (solo install, legacy). Bot comments post under this PAT's owner.
+GH_TOKEN=
+GITHUB_TOKEN=  # Same as GH_TOKEN, used by the GitHub adapter in PAT mode.
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)